app.js

```diff
@@ -64,21 +64,70 @@ function catchError(err, res) {
   res.status(500).end();
 }
 
+function generateTestPage(endpoint) {
+  const theseTests = tests.getTests(endpoint);
+  let testScope = tests.getScope(endpoint);
+  let individual = false;
+
+  const lines = [
+    '<!DOCTYPE html>',
+    '<html>',
+    '<head>',
+    '<!--Copyright 2020 Google LLC',
```

Comment:
Let's drop this. A big comment at the top is annoying when viewing source to understand the test. If I hear that we need this even for generated output then we can add it back at the bottom.

Reply:
Okay, I'll go ahead and drop it!  I think it'll be all fine, since there's copyright notices elsewhere.